config/psyclone.cfg

```diff
@@ -75,6 +75,46 @@ access_mapping = read: read, write: write, readwrite: readwrite,
 [gocean1.0]
 access_mapping = go_read: read, go_write: write, go_readwrite: readwrite
 
+# The detail for grid properties. There are three fields:
+# 1) the name used in a kernel declaration
+# 2) The string required to dereference the property from the field.
+#    A {0} is replaced with the name of the field variable
+# 3) The type of the property, must be 'scalar' or 'array'
+# The following values are used by PSyclone, and are therefore required:
+# go_grid_xstop, go_grid_ystop, 
+# go_grid_{internal,external}_{inner,outer}_{start,stop}
```

Comment:
_upper_bound etc. It is assembled in various if-statements, so you won't find any of the
terms directly with a grep. I am not sure if we should change this as part of this request, it could break existing code if some other functionality (e.g. in a user code) should use one of the names? Admittedly unlikely though, but I would prefer to add this to #639 - I think it might be worth having a closer look at the existing properties (see #639).

Reply:
I've added this together with your comment for the next issue to #639. If you prefer, I can fix this part earlier of course.